### PR TITLE
Fix/day picker

### PR DESCRIPTION
**Pull request by vozbrann**

### CODE REVIEW

File: src/components/form/day-picker/index.tsx [hunk 1]
```diff
@@ -115,12 +122,12 @@ export const DayPicker = ({
 
   React.useEffect(() => {
     const close = (e) => {
-      if(e.keyCode === 27 || e.keyCode === 13){
+      if (e.keyCode === 27) {
```
Comment (doichev-kostia) [inline src/components/form/day-picker/index.tsx:125]: I'd write a comment with a definition of the key, which code is 27, as it's a bit cryptic

```typescript
// key with code 27 is "Escape"
if (e.keyCode === 27){
closeCalendar();
}
```

File: src/components/form/day-picker/index.tsx [hunk 2]
```diff
@@ -136,87 +143,92 @@ export const DayPicker = ({
...
+  };
 
   const CalendarComponent = () => (
     <div className='footer'>
       <MaskedInput
-        id="my-input-id"
-        key="my-input-id"
+        id='my-input-id'
+        key='my-input-id'
         render={(customRef, restProps): JSX.Element => (
```
Comment (doichev-kostia) [inline src/components/form/day-picker/index.tsx:153]: I'd add types here

Comment (doichev-kostia) [reply src/components/form/day-picker/index.tsx:153]: or they are inferred?

**Review by doichev-kostia — APPROVED** [review]
Just add small changes

**Review by Yhuley — APPROVED** [review]

**Review by OlhaNikoliuk — APPROVED** [review]

**Review by vlacher12 — APPROVED** [review]

**Pull request merged**